[IMP] empty

arpi-odoo opened the pull request.

arpi-odoo committed:
[FIX] hr: new contract button invisible

arpi-odoo committed:
[IMP] hr_skills: cleanup certification reporting

arpi-odoo committed:
[IMP] hr_payroll: constraint positive wage

arpi-odoo committed:
[FIX] hr_holidays: fix traceback leave type

arpi-odoo committed:
[FIX] hr: fix delete version

arpi-odoo committed:
[FIX] hr: fix various bugs

Mohamed-Khaled308 committed:
[FIX] hr: fix new contract button visibility

Made the new contract button invisible when no contract start date

Task 5231718

arpi-odoo committed:
[IMP] empty

robodoo commented:
This PR targets the un-managed branch odoo-dev/odoo:master-hr-onboarding, it needs to be retargeted before it can be merged.

Mohamed-Khaled308 committed:
[FIX] hr_holidays: fix time off type error in time off form view

1.  Go to **Time Off > Configuration > Time Off Types**.
2.  Open a type and click the "Time Off" smart button.
3.  Click **New**.
4.  An error appears.

[[Reference Video](https://drive.google.com/file/d/1vi_apL24Km5tSomQtwcTArqqWvrBR5ni/view)]

In the `_search_virtual_remaining_leaves` method, the code tries to use an operator (like `>`) to compare the `virtual_remaining_leaves`.

This operator requires **two** arguments to work. However, the code was only passing **one** argument, missing the `value` to compare against. This caused the "expects 2 arguments" error.

Passed the missing `value` to the operator call so the comparison can be completed correctly.

Task: 5241685
